src/devices/Bmi088/GyroscopeOutputDataRate.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Iot.Device.Bmi088Device
+{
+    /// <summary>
+    /// Data rates for the gyroscope
+    /// </summary>
+    public enum GyroscopeOutputDataRate : byte
+    {
+        /// <summary>See datasheet</summary>
+        Odr2000_Bw532 = 0x00,
```

Comment:
There should not be any underscore. Also, better to use Bandwidth than Bw. Odr is fine in this case because it is present in the enum name and you can't start an enum with a number.